### PR TITLE
Update import scripts to work with 15 Feb 2015 dump

### PR DESCRIPTION
**Pull request by dgorissen**

### CODE REVIEW

File: manage.py [hunk 1]
```diff
@@ -118,7 +118,7 @@ def print_flush(msg):
     print_every = 1000
     print_flush("Adding waterpoints. Please be patient.")
 
-    with open(filename) as f:
+    with open(filename, 'rU') as f:
```
Comment (kynan) [inline manage.py]: What are the line terminators for this file? Not opposed to this change, just curious.

File: scripts/openrefine.json [hunk 2]
```diff
@@ -1,4 +1,4 @@
-i[
```
Comment (kynan) [inline scripts/openrefine.json]: did this ever work?

**Comment by kynan** [comment]
Looks good to me. I assume you tested this @dgorissen?

File: manage.py (hunk 1 above)
Comment (dgorissen) [inline manage.py]: This is just to make life easier when exporting to csv from Excel (what you typically do as part of the workflow). Adding 'U' ensures those csv's can be easily read too.

File: scripts/openrefine.json (hunk 2 above)
Comment (dgorissen) [inline scripts/openrefine.json]: mm typo, well spotted

**Comment by dgorissen** [comment]
its whats running on taarifa-dev

File: manage.py (hunk 1 above)
Comment (kynan) [inline manage.py]: OK, happy with that.

File: scripts/openrefine.json (hunk 2 above)
Comment (kynan) [inline scripts/openrefine.json]: So this did indeed never work? Should maybe have a test for it.

**Pull request merged**